src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -328,7 +367,8 @@ define(function (require, exports, module) {
         // TinyColor actually generates to see if it's different. If so, then we assume the color
         // was incomplete to begin with.
         if (newColorOk) {
-            newColorOk = (newColorObj.toString() === this._normalizeColorString(newColor));
+            var colorStr = newColor.replace("0x","#") || newColor;
```

Comment:
this `newColor.replace("0x","#") || newColor` should be wrapped in a function with a proper name like `normalizeColorString` or something which hints, why we're doing this